frontend/src/components/Psychologist/Profile/ConventionForm.jsx

```diff
@@ -56,6 +59,13 @@ const ConventionForm = ({ currentConvention, onConventionUpdated, checkDefaultVa
               ? universities.map(university => ({ value: university.id, label: university.name }))
               : []}
           />
+          {((laSorbonneParisNord && convention.universityId === laSorbonneParisNord.id) || (laSorbonne && convention.universityId === laSorbonne.id)) && (
```

Comment:
I would have extract this part in a const called for example `isSorbonne`

Reply:
I don't think it's the best option, everywhere we did it directly in the return. Maybe if we do this here, we have to do it everywhere then :) But if u think it's more visible and efficient, i will do it 👍

Reply:
we discuss ( she gave me good arguments ) and i will do it :)

Reply:
Done 🥇